xfel/command_line/cspad_cbf_metrology.py

```diff
@@ -67,7 +67,7 @@
     .type = choice
     .help = Algorithm to be used for choosing the n_subset images/experiments for \
             refinement.  n_refl chooses the set with the largest numbers of reflections \
-            listed in the pickle files, thus giving maximal coverage of the detector tiles \
+            listed in the mpack files, thus giving maximal coverage of the detector tiles \
```

Comment:
suggest using "reflection table" for things like this, then it's independent of whatever format